### PR TITLE
Future - cleanup up dependencies

### PR DESCRIPTION
**Pull request by ndelangen**
Remove dependencies not used for core-common,
because most of this was removed with babelModeV7 or was moved to presets

### CODE REVIEW

**Comment by ndelangen** [comment]
Hmm I really really want to remove the regenerator runtime!

I know react-inspector needs it at runtime. I could invest some time updating that to tsup? Or we could switch to other library we found a while ago @yannbf

**Comment by ndelangen** [comment]
I think I could also add the regenerator runtime like a global via lib/ui!
That would at least be better and place the dependency closest to where it's actually used. And clean up where dependencies come from.

**Comment by ndelangen** [comment]
Scrap that, I just upgraded react-inspector to output modern code

File: scripts/bootstrap.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ function run() {
       option: '--prep',
       command: () => {
         log.info(prefix, 'prepare');
-        spawn(`nx run-many --target="prepare" --all --parallel -- --reset`);
+        spawn(
+          `nx run-many --target="prepare" --all --parallel --exclude="@storybook/addon-storyshots" -- --reset`
```
Comment (shilman) [inline scripts/bootstrap.js:70]: what's going on here? work in progress?

**Pull request closed without merging**